services/php-fpm/Dockerfile

```diff
@@ -1,4 +1,6 @@
-ENV COMPOSER_NO_INTERACTION=1
+ENV COMPOSER_DISCARD_CHANGES=true
```

Comment:
Should there be a `php-base` docker image that we can `FROM` from to ensure we have the same environment variables regardless of SAPI type?

Reply:
I actually looked for a PHP base when I went to do this, so that would be welcome, but non-blocking I think.

Reply:
Agreed!